packages/nimble-components/src/combobox/testing/combobox.pageobject.ts

```diff
@@ -0,0 +1,128 @@
+import { keyEnter, keyArrowDown } from '@microsoft/fast-web-utilities';
+import type { Combobox } from '..';
+import { waitForUpdatesAsync } from '../../testing/async-helpers';
+import {
+    createEventListener,
+    waitAnimationFrame
+} from '../../utilities/tests/component';
+
+/**
+ * Page object for the `nimble-combobox` component to provide consistent ways
+ * of querying and interacting with the component during tests.
+ */
+export class ComboboxPageObject {
+    private readonly regionLoadedListener = createEventListener(
+        this.comboboxElement,
+        'loaded'
+    );
+
+    public constructor(protected readonly comboboxElement: Combobox) {}
+
+    public async waitForAnchoredRegionLoaded(): Promise<void> {
+        await this.regionLoadedListener.promise;
+    }
```

Comment:
thinking of the pageobject as abstracting details, this one feels strange to be in the public api. Could it instead be private and used in apis like clickAndWaitForOpen?

Reply:
I'm already waiting for the region loaded promise in the two click-and-wait-for-open functions, but if I also wait for it in the commit function, I think that works.